src/os_win32.c

```diff
@@ -232,6 +236,14 @@ static char_u *exe_path = NULL;
 
 static BOOL win8_or_later = FALSE;
 
+# if defined(__GNUC__) && !defined(__MINGW32__)  && !defined(__CYGWIN__)
+#  define AChar AsciiChar
```

Comment:
It seems that `AChar` is not used.

Reply:
Perhaps it should be used. I will fix it from now.